src/ReturnBookUi.java

```diff
@@ -1,22 +1,47 @@
 import java.util.Scanner;
 
-public class ReturnBookUI {
+public class ReturnBookUi {
 
-    public static enum UI_STATE {INITIALISED, READY, INSPECTING, COMPLETED};
+    public static enum UiState { INITIALISED, READY, INSPECTING, COMPLETED };
     
     private ReturnBookControl control;
+    
     private Scanner input;
-    private UI_STATE state;
+    
+    private UiState state;
 
     
-    public ReturnBookUI(ReturnBookControl control) {
+    public ReturnBookUi(ReturnBookControl control) {
         this.control = control;
         input = new Scanner(System.in);
-        state = UI_STATE.INITIALISED;
-        control.setUI(this);
+        
+        state = UiState.INITIALISED;
+        
+        control.setUi(this);
+    }
+    
+    
+    private String getInput(String prompt) {
+        System.out.print(prompt);
+        return input.nextLine();
+    }
+
+    
+    private void output(Object object) {
+        System.out.println(object);
     }
 
     
+    public void display(Object object) {
+        output(object);
+    }
+
+    
+    public void setState(UiState state) {
+        this.state = state;
+    }
+    
+    
     public void run() {
         output("Return Book Use Case UI\n");
         while (true) {
```

Comment:
D15 - Add a new line in between lines 48 and 49 to make spacing uniform throughout file.